selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php

```diff
@@ -9,6 +9,8 @@
  * Contributors:
  *   Red Hat, Inc. - initial API and implementation
  */
+
 function sayHello($name) {
-    return "Hello, " + $name;
+    return "Hello, $name";
```

Comment:
It's interesting to know why did you decided to change the code.

Reply:
http://php.net/manual/en/language.operators.string.php

It can be:
```
"Hello, $name"
"Hello, ".$name
```

Reply:
I see, but you decided to change previous variant from some reasons, didn't it :-)

Reply:
because `+` doesn't concatenate stings in php.

Reply:
Hm. it explains everything. Good catch!